lib/patch_ruby/models/order.rb

```diff
@@ -221,7 +230,7 @@ def valid?
       return false if @mass_g < 0
       return false if @production.nil?
       return false if @state.nil?
-      state_validator = EnumAttributeValidator.new('String', ["draft", "placed", "complete", "cancelled"])
+      state_validator = EnumAttributeValidator.new('String', ["draft", "placed", "processing", "complete", "cancelled"])
```

Comment:
@bspellacy take-away is that we need to update those SDKs more often. Long-term I'm wondering if we could get to automate the submission of PRs upon updates to the Swagger file.

Reply:
cc @kleinjm